solr/conf/solrconfig.xml

```diff
@@ -91,7 +91,7 @@
          normalized_title_teim^100
          name_teim^10
          place_teim^10
-         unitid_identifier_match^5
```

Comment:
small thing, but could we bump this up above name/place? it's nice to have them sorted by boost in descending order

Reply:
to clarify: just shift the line upwards, but keep the same boosting, so that the boost fields are sorted

Reply:
@thatbudakguy good idea, done!